erlangen_nuernberg.py

```diff
@@ -0,0 +1,234 @@
+                type += 'Geflügel '
+            if('V' in td.string):
+                type += 'Vegetarisch '
+            if('F' in td.string):
+                type += 'Fisch '
+            if('L' in td.string):
+                type += 'Lamm '
+            if('W' in td.string):
+                type += 'Wild '
+        return type
+
+    def get_refs(td):
+        return td.find_all('sup')
+
+    def get_foot_description(td):
+        refl = get_refs(td)
+        description = td.text
+        for ref in refl:
+            description = description.replace(' '+ref.text, '', 1)
+        if description[0] == ' ':
+            description = description.replace(' ', '', 1)
+        return description
+
+    def get_notes(td):
+        refl = get_refs(td)
+        strl = []
+        for ref in refl:
+            strl.extend(ref.string.split(','))
+        strl = list(set(strl))
+        return strl
+
+    def build_notes_string(td):
+        refs = get_notes(td)
+        food_is = ''
+        food_contains = ''
+        for r in refs:
+            # parse food is footnotes
+            if r == '1':
+                food_is += 'mit Farbstoffen, '
+            elif r == '4':
+                food_is += 'geschwärzt, '
+            elif r == '7':
+                food_is += 'mit Antioxidationsmittel, '
+            elif r == '8':
+                food_is += 'mit Geschmacksverstärker, '
+            elif r == '9':
+                food_is += 'geschwefelt, '
+            elif r == '10':
+                food_is += 'geschwärzt, '
+            elif r == '11':
+                food_is += 'gewachst, '
+            elif r == '12':
+                food_is += 'mit Phosphat, '
+            elif r == '5':
+                food_is += 'mit Süßungsmittel, '
+            # parse allergic footnotes
+            elif r == 'a1':
+                food_contains += 'Gluten, '
+            elif r == 'a2':
+                food_contains += 'Krebstiere, '
+            elif r == 'a3':
+                food_contains += 'Eier, '
+            elif r == 'a4':
+                food_contains += 'Fisch, '
+            elif r == 'a5':
+                food_contains += 'Erdnüsse, '
+            elif r == 'a6':
+                food_contains += 'Soja, '
+            elif r == 'a7':
+                food_contains += 'Milch/Laktose, '
+            elif r == 'a8':
+                food_contains += 'Schalenfrüchte, '
+            elif r == 'a9':
+                food_contains += 'Sellerie, '
+            elif r == 'a10':
+                food_contains += 'Senf, '
+            elif r == 'a11':
+                food_contains += 'Sesam, '
+            elif r == 'a12':
+                food_contains += 'Schwefeldioxid/Sulfite, '
+            elif r == 'a13':
+                food_contains += 'Lupinen, '
+            elif r == 'a14':
+                food_contains += 'Weichtiere, '
+            else:
+                food_contains += 'undefinierte Chemikalien:'+r+', '
```

Comment:
Could you extract this data from the website - so updates would be found automatically?